### ISSUE TITLE
[Assigned] Save searched words history.

### ISSUE THREAD

**Issue by konifar** [DroidKaigi/conference-app-2018#395]
## Overview (Required)
- Title says everything.
- Like Google I/O app, Twitter, or other apps.

**Comment by TomoyaShibata** [comment]
I will try this! 🙋

**Comment by takahirom** [comment]
Thanks! Assigned 👍

**Comment by TomoyaShibata** [comment]
Successfully saved search words and displayed suggestions.  
 https://github.com/TomoyaShibata/conference-app-2018/commit/e2c23275836868a2a70e59a4a11ddde69a41f3ac

<img src="https://user-images.githubusercontent.com/2193123/35371337-d9f5aa78-01d6-11e8-8dfb-0225a48bee56.png" width="300" />

But it doesn't match the design of the application. (For example, clock icon)
I challenged to change the `styles.xml`, but I failed :persevere:

If you know someone who knows it, I want you to help me.
